libbpf-tools/filetop.bpf.c

```diff
@@ -0,0 +1,91 @@
+/* SPDX-License-Identifier: (LGPL-2.1 OR BSD-2-Clause) */
+/* Copyright (c) 2021 Hengqi Chen */
+#include <vmlinux.h>
+#include <bpf/bpf_helpers.h>
+#include <bpf/bpf_core_read.h>
+#include <bpf/bpf_tracing.h>
+#include "filetop.h"
+#include "stat.h"
+
+#define MAX_ENTRIES	10240
+
+const volatile pid_t target_pid = 0;
+const volatile bool full_path = false;
```

Comment:
This isn't being used anywhere. Same comment applies to `show_full_path` flag setting in the userspace runner.

Reply:
Will remove this for now and add this option once bpf_d_path helper expose to vfs_*